dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -79,7 +95,10 @@
       - location = Geocoder.search(request.ip).try(:first)
       - country_code = location&.country_code.to_s.upcase
       = f.label :us_state, t('activerecord.attributes.user.us_state'), class: 'label-bold'
-      = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, required: true
+      - if studentInLockoutFlow
+        %em= t('user.update_field_parent_permission_required')
+        %br/
+      = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, disabled: studentInLockoutFlow, required: true
```

Comment:
```suggestion
      - if !isUSA
        = f.select :us_state, us_state_options, {include_blank: t('school_info.select_state')}, disabled: studentInLockoutFlow, required: true
```
This was missed in the original PR where the field was added, but we shouldn't show this input to students outside the USA.

Reply:
The logic feels weird... why is it `!isUSA` ... shouldn't it be `if isUSA` to render the state selection?